src/views/shared/Footer.js

```diff
@@ -21,10 +21,14 @@ function (Okta) {
 
   return Okta.View.extend({
     template: '\
-      <a href="#" data-se="needhelp" class="link help js-help">\
+      <a href="#" \
```

Comment:
nit: maybe just keep this on two lines. I don't think we need a line for each attribute